boomkat_importer.user.js

```diff
@@ -0,0 +1,162 @@
+// ==UserScript==
+// @name        Import Boomkat releases to Musicbrainz
+// @description Add a button on Boomkat release pages to open MusicBrainz release editor with pre-filled data for the selected release
+// @version     2020.07.21.1
+// @namespace   https://github.com/murdos/musicbrainz-userscripts
+// @include     https://boomkat.com/products/*
+// @require     https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js
+// @require     lib/mbimport.js
+// @require     lib/logger.js
+// @require     lib/mbimportstyle.js
+
+// ==/UserScript==
+
+// prevent JQuery conflicts, see http://wiki.greasespot.net/@grant
+this.$ = this.jQuery = jQuery.noConflict(true);
+
+if (!unsafeWindow) unsafeWindow = window;
+
+$(document).ready(function () {
+    MBImportStyle();
+
+    let release_url = window.location.href.replace('/?.*$/', '').replace(/#.*$/, '');
```

Comment:
Since you never change the value of this variable, you should use `const` instead of `let`.
This is true for most usage of `let` in this script.